charts/dkube-deployer/values.yaml

```diff
@@ -126,6 +126,77 @@ optional:
         #To enable modelmonitor in dkube. (true / false)
         enabled: "false"
 
+    
+    promscale:
+        #To enable promscale in dkube. (true / false)
+        enabled: "true"
+
+        timescaledb:
+            # to use external timescale db
+            external: "false"
+            
+            # keep it false if db is already existing
+            # create: false
+
+            # timescale db | default: timescale
+            db: ""
+
+            # database user | default: postgres
+            user: ""
```

Comment:
these are all external or internal also? Can we group all external and backup items together either with a prefix or using a separate section?

Reply:
Its for the case if timescale is internal and backup source we want to use external
Earlier i have written them in external section only but that external section was only for promscale so i thought user will think that backup will belong to promscale